frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -164,12 +167,22 @@ const mapStateToProps = (state: OxalisState): StateProps => {
   const isVisibleButUneditableSegmentationLayerActive =
     visibleSegmentationLayer != null && visibleSegmentationLayer.tracingId == null;
 
+  const addCoordString = getAdditionalCoordinatesAsString(state.flycam.additionalCoordinates);
+
+  let meshesForCurrentAdditionalCoordinates = EMPTY_OBJECT;
+  if (visibleSegmentationLayer != null) {
+    const meshRecords = state.localSegmentationData[visibleSegmentationLayer?.name].meshes;
+    meshesForCurrentAdditionalCoordinates =
+      meshRecords != null &&
+      Object.keys(meshRecords).length > 0 &&
+      Object.keys(meshRecords[addCoordString]).length > 0
+        ? meshRecords[addCoordString]
+        : EMPTY_OBJECT;
+  }
```

Comment:
I think this could be extracted into an `getMeshesForAdditionalCoordinates()` function that could be put into the volumetracing_accessor.ts module.